tests/test_places.py

```diff
@@ -0,0 +1,121 @@
+from apistar.test import TestClient
+from app import app
+import pytest
+import json
+import os
+
+"""
+    This module tests basic query against the endpoint '/places/'
+"""
+
+def load_place(file_name, mimir_client, index_name, _doc_type):
+    """
+    Load a json file in the elasticsearch and returns the corresponding Place id
+    """
+    filepath = os.path.join(os.path.dirname(__file__), 'fixtures', file_name)
+    with open(filepath, "r") as f:
+        place = json.load(f)
+        place_id = place['id']
+        mimir_client.index(index=index_name,
+                        body=place,
+                        doc_type=_doc_type, # 'admin', 'street', 'addr' or 'poi'
+                        id=place_id,
+                        refresh=True)
+        return place_id
+
+@pytest.fixture(scope="session")
+def admin_goujounac(mimir_client, init_indices):
+    """
+    fill elasticsearch with an admin
+    """
+    return load_place('admin_goujounac.json', mimir_client, 'munin_admin_test', 'admin')
+
+@pytest.fixture(scope="session")
+def street_birnenweg(mimir_client, init_indices):
+    """
+    fill elasticsearch with a street
+    """
+    return load_place('street_birnenweg.json', mimir_client, 'munin_street_test', 'street')
+
+@pytest.fixture(scope="session")
+def address_du_moulin(mimir_client, init_indices):
+    """
+    fill elasticsearch with an address
+    """
+    return load_place('address_du_moulin.json', mimir_client, 'munin_addr_test', 'addr')
+
+@pytest.fixture(scope="session")
+def orsay_museum(mimir_client, init_indices):
+    """
+    fill elasticsearch with the orsay museum
+    """
+    return load_place('orsay_museum.json', mimir_client, 'munin_poi', 'poi')
+
+
+def test_basic_query_admin(admin_goujounac):
```

Comment:
It's seems to work but I'm not a fan of this mechanism, you create side effects by loading stuff in ES and not cleaning afterward.
if you run this test first then `test_basic_query_street`, in the second test, ES will have `admin_goujounac` and `street_birnenweg`.

either always add all 3 objects of clean afterward*.

Side effects in tests can create very hard to debug king of bugs

Reply:
done